include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -238,7 +239,7 @@ ORT_API_STATUS(OrtSetSessionLogVerbosityLevel, _In_ OrtSessionOptions* options,
 ORT_API_STATUS(OrtSetSessionGraphOptimizationLevel, _In_ OrtSessionOptions* options, uint32_t graph_optimization_level);
 
 // How many threads in the session thread pool.
-ORT_API_STATUS(OrtSetSessionThreadPoolSize, _In_ OrtSessionOptions* options, int session_thread_pool_size);
+ORT_API_STATUS(OrtSetSessionThreadPoolSize, _In_ OrtSessionOptions* options, unsigned int session_thread_pool_size);
```

Comment:
Not every language has unsigned int,e.g. java  It will be harder to invoke this function in java.

Reply:
I saw 2 other places where we're using unsigned int because the underlying data type was that. Let me use uint32_t instead. I don't want to use int as that would mean values can be negative.

ORT_API_STATUS(OrtRunOptionsSetRunLogVerbosityLevel, _In_ OrtRunOptions*, unsigned int);
ORT_API_STATUS(OrtRunOptionsGetRunLogVerbosityLevel, _In_ OrtRunOptions*, _Out_ unsigned int* out);

Reply:
Can we just int? For example,  Java doesn't have unsigned integer types.

Reply:
Ok, using ints now.